pkg/lib/operation/template/local/test/create/operation.go

```diff
@@ -0,0 +1,90 @@
+package create
```

Comment:
@webrouse Prosím tě já tady mám problém s nějakým import cyklem a nedaří se mi zjistit co mu vadí a jak to fixnout. Můžeš mě nějak nasměrovat? 🙂

Chápu to tak, že se mu nelíbí, že sem importuju `github.com/keboola/keboola-as-code/internal/pkg/template/test` a v něm se importuje dialog? (https://github.com/keboola/keboola-as-code/blob/bc8a088e5e6ece268b9fc0c9ac7f6fa1f2a2116c/internal/pkg/template/test/template.go#L8)

Zvláštní ale je, že když udělám checkout druhýho commitu (https://github.com/keboola/keboola-as-code/pull/783/commits/05e35e161e4c3c6836ff067a0d7ce0fdca3a1492), tak tam ten problém není. - T.j. že mu to nevadí v `operation/template/local/test/run/operation` kde se ten `internal/pkg/template/test` importuje a taky používá téměř stejně (https://github.com/keboola/keboola-as-code/blob/bc8a088e5e6ece268b9fc0c9ac7f6fa1f2a2116c/pkg/lib/operation/template/local/test/run/operation.go#L20)

![CleanShot 2022-09-01 at 14 40 31](https://user-images.githubusercontent.com/215660/187916053-2caa53bd-7d98-4996-b36a-219b0f06b6a3.jpg)

Reply:
Problem bude asi v tom ako v testoch riesis zavislosti:
![image](https://user-images.githubusercontent.com/19371734/187920678-28ad94cc-743b-4128-ab1d-ab6e7c8333f4.png)

Na ziadnu operaciu by si nemal potrebovat dialog, .... dialog sa vola vzdy priamo z CLI commandu na ziskanie nejakych informacii, a potom sa spusti operacia, tym by ti nemali vzniknut ziadne cyklicke zavislosti.

Vid napr `init` cmd:
https://github.com/keboola/keboola-as-code/blob/a4908a90d98b0e48a677c3ce201b59f89b86d679/internal/pkg/cli/cmd/sync/init.go#L28-L32

Reply:
To aby si tam nemal zavislost na `dialogs` som uz vyriesil v: https://github.com/keboola/keboola-as-code/pull/782
(riesim uz iba jeden padajuci test na Windows a je to komplet).

Reply:
Za chvilu na to pozriem podrovnejsie, co na com zavisi ...

Reply:
To chápu no, akorát já tam potřebuju teda nějak sestavit ty závislosti pro templatu (https://github.com/keboola/keboola-as-code/blob/05e35e161e4c3c6836ff067a0d7ce0fdca3a1492/internal/pkg/template/test/template.go#L27-L28) a projekt (https://github.com/keboola/keboola-as-code/blob/05e35e161e4c3c6836ff067a0d7ce0fdca3a1492/internal/pkg/template/test/project.go#L56-L57)

Reply:
Ak by to nepomohlo tak spolu to vymyslime.

Reply:
Konkretne je v tej 782 toto:
https://github.com/keboola/keboola-as-code/blob/ea5e0a794dc8e047516f850fa22edf771ac69fe7/pkg/lib/operation/template/local/test/operation.go#L128

https://github.com/keboola/keboola-as-code/blob/ea5e0a794dc8e047516f850fa22edf771ac69fe7/pkg/lib/operation/template/local/test/operation.go#L413-L434